modules/imgproc/src/ccl_bolelli_forest.inc.hpp

```diff
@@ -0,0 +1,1783 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
+//
+//  IMPORTANT: READ BEFORE DOWNLOADING, COPYING, INSTALLING OR USING.
+//
+//  By downloading, copying, installing or using the software you agree to this license.
+//  If you do not agree to this license, do not download, install,
+//  copy or use the software.
+//
+//
+//                        Intel License Agreement
+//                For Open Source Computer Vision Library
```

Comment:
Please use short license header: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
//
// 2021 Federico Bolelli <federico.bolelli@unimore.it>
// 2021 Stefano Allegretti <stefano.allegretti@unimore.it>
// 2021 Costantino Grana <costantino.grana@unimore.it>
```

Reply:
@alalek, sorry I've missed it. Do you want me to replace also the [connectedcomponents.cpp](https://github.com/prittt/opencv/blob/sota-ccl/modules/imgproc/src/connectedcomponents.cpp) license header or just the ones in the hpp files?

Reply:
Lets update in new files only for now.